Applications/Api_example/jni/main.cpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2024 SeungBaek Hong <sb92.hong@samsung.com>
+ *
+ * @file   main.cpp
+ * @date   15 March 2024
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author SeungBaek Hong <sb92.hong@samsung.com>
+ * @bug	   No known bugs except for NYI items
+ *
+ * @note This is experimental API and not stable.
+ */
+
+#include <functions.h>
+#include <iostream>
+
+using namespace ml::train;
+
+Tensor& build_model(Tensor &x) {
```

Comment:
How about doing it this way?

```
Tensor& build_model(Tensor &x) {
  x = pow(pow(pow(x)));
  return x;
}
```
or 
```
Tensor& build_model(Tensor &x) {
  return pow(pow(pow(x)));
}
```

Reply:
That seems to be a good coding style. However, this example was intentionally coded using line changes to show that variables can be reused and layers can be stacked using these variables. thanks :)